### PR TITLE
Removed reference to issue and uncommented VCH Create Test

### LINKED ISSUE #384: Docker API: Container Wait
Implement the docker wait operation in the engine-api server and integrate with interaction port layer.

https://docs.docker.com/engine/reference/api/docker_remote_api_v1.22/#wait-a-container
https://docs.docker.com/engine/reference/commandline/wait/

Please note, the engine-api's rest server already packaging these json struct into golang structs. The url above is just for reference on what the operation should do.

### PR DESCRIPTION
**Pull request by cgtexmex**
VCH Create test uncommented and will now verify the size of a custom base disk.

fixes #384

### CODE REVIEW

**Comment by cgtexmex** [comment]
@gigawhitlocks @mhagen-vmware @andrewtchin please review this...not sure the reasoning for the docker commands on this one test.

File: tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot [hunk 1]
```diff
@@ -19,14 +19,6 @@ Create VCH - custom base disk
     Get Docker Params  ${output}  ${true}
     Log To Console  Installer completed successfully: ${vch-name}...
 
-    ${status}=  Get State Of Github Issue  384
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-6-Docker-Run.robot needs to be updated now that Issue #384 has been resolved
-    # Replace this hack when we have docker run in a state where it's usable in tests (Issue #384)
-
-    ${output}=  Run  docker ${params} logs $(docker ${params} start $(docker ${params} create busybox /bin/df -h) && sleep 10) | grep /dev/sda | awk '{print $2}'
```
Comment (andrewtchin) [inline tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot]: it looks like this line is supposed to check that the base image size is correctly used. is there another way to test that size?

Comment (mhagen-vmware) [inline tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot]: It is definitely intended to confirm the size requested. Is there a reason why you are removing it?

Comment (cgtexmex) [inline tests/test-cases/Group6-VIC-Machine/6-4-Create-Basic.robot]: I was confused -- first buy the fact that it was referencing run and second by the nature of the command.  I just ran it locally and it worked fine.....so I've put it back :)

**Comment by andrewtchin** [comment]
lgtm

**Comment by jzt** [comment]
lgtm

**Pull request merged**